presto-hive/src/main/java/com/facebook/presto/hive/util/Statistics.java

```diff
@@ -148,6 +470,8 @@ private static long convertLocalToUtc(DateTimeZone timeZone, long value)
     {
         ADD,
         SUBTRACT,
+        SELECT_MIN,
+        SELECT_MAX,
```

Comment:
nit: why not just "MIN", "MAX"

Reply:
So i can static-import it. `MIN` and `MAX` are used by the values from the `ColumnStatisticType`

Reply:
Going to rename it back to `MIN` and `MAX` after i rename values in `ColumnStatisticType`